RUFAS/routines/feed_storage/enums.py

```diff
@@ -33,35 +33,35 @@ class CropType(Enum):
     Attributes
     ----------
     WHEAT : str
-        Type of Small Grain.
+        Represents a type of type of Small Grain.
```

Comment:
```suggestion
        Represents a type of Small Grain.
```
Same comment for other attributes in this docstring.